doc/src/explanation/active-deprecations.md

```diff
@@ -78,26 +78,28 @@ SymPy deprecation warnings.
 
 (moved-mechanics-functions)=
 ### Moved mechanics functions
-With the introduction of some new objects like the ``Inertia`` object in the
-``sympy.physics.mechanics`` module, some functions from
+With the introduction of some new objects like the ``Inertia`` and load objects
```

Comment:
Does "load" need to be formatted as code? Or is it generally referring to multiple load-related objects?

Reply:
It is generally referring to multiple load-related objects.